app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_parts = command.split()
```

Comment:
The variable name `command_parts` could be more descriptive. Consider renaming it to something more meaningful, like `parsed_command`.